cmd/state-installer/installer.go

```diff
@@ -235,6 +235,10 @@ func installedOnPath(installRoot, channel string) (bool, string, error) {
 		return false, "", nil
 	}
 
+	if !fileutils.FileExists(filepath.Join(installRoot, installation.InstallDirMarker)) {
+		return false, "", nil
+	}
```

Comment:
What I meant with the AC is to return True if this was encountered. So add it to the candidates below.

I think we're avoiding asserting false if it didn't exist in order to support older state tool versions that didn't supply this marker.